spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniparc/converter/UniParcDocumentConverter.java

```diff
@@ -35,7 +36,11 @@ private void processDbReference(
             UniParcCrossReference xref, UniParcDocument.UniParcDocumentBuilder builder) {
         UniParcDatabase type = xref.getDatabase();
 
-        builder.dbId(xref.getId());
+        if (Objects.nonNull(xref.getVersion())) {
+            builder.dbId(xref.getId() + "." + xref.getVersion());
+        } else {
+            builder.dbId(xref.getId());
```

Comment:
We always have to have the id without version. if there is a version, we have to index with the version as well

Reply:
Thanks.. added.